Packs/DefaultPlaybook/Playbooks/Default_8_2.yml

```diff
@@ -0,0 +1,1195 @@
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "68":
+    id: "68"
+    taskid: b75662a2-5440-4d64-8635-47c4f552b02d
+    type: regular
+    task:
+      id: b75662a2-5440-4d64-8635-47c4f552b02d
+      version: -1
+      name: Review before closing the incident
+      description: Ensure the incident was properly remediated and report the incident if needed, before closing it.
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "41"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 452.5,
+          "y": 3870
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "69":
+    id: "69"
+    taskid: 153f8c88-6fc3-446c-8fb8-822e1cdcf129
+    type: playbook
+    task:
+      id: 153f8c88-6fc3-446c-8fb8-822e1cdcf129
+      version: -1
+      name: Get File Sample - Generic
+      description: Retrieve files from endpoints by the file hash or the file path.
+      playbookName: Get File Sample - Generic
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "19"
+    scriptarguments:
+      AgentID:
+        complex:
+          root: incident
+          accessor: agentid
+      FilePath:
+        complex:
+          root: incident
+          accessor: filepath
+      Hostname:
+        complex:
+          root: incident
+          accessor: sourcehostname
+      MD5:
+        complex:
+          root: File
+          accessor: MD5
+      SHA256:
+        complex:
+          root: File
+          accessor: SHA256
+      UnzipFile:
+        simple: "True"
+    separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 0
+    view: |-
+      {
+        "position": {
+          "x": 52.5,
+          "y": 950
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "39_16_#default#": 0.23,
+      "42_45_#default#": 0.58,
+      "42_58_false positive": 0.42,
+      "45_58_False Positive": 0.48,
+      "7_35_Yes": 0.21
+    },
+    "paper": {
+      "dimensions": {
+        "height": 4480,
+        "width": 1620,
+        "x": -470,
+        "y": 100
+      }
+    }
+  }
+inputs:
+- key: ExtractIndicators
+  value:
+    simple: "True"
+  required: false
+  description: Determines whether indicators in this incident should be extracted manually. If you turned off Auto-extract for the current incident type, you may want to set this value to True. Otherwise indicators from this incident will not be extracted, which will negatively affect the efficacy of the playbook.
```

Comment:
```suggestion
  description: Determines whether indicators in this incident should be extracted manually. If you turned off Auto-extract for the current incident type, you may want to set this value to True. Otherwise, indicators from this incident will not be extracted, which will negatively affect the efficacy of the playbook.
```